mariadb.spec

```diff
@@ -755,6 +777,7 @@ rm -r storage/tokudb/mysql-test/tokudb/t/*.py
 
 
 %build
+%{set_build_flags}
```

Comment:
This macro does not exist in centos7, build is failing:

+ cd mariadb-10.3.20
RPM build errors:
+ '%{set_build_flags}'
/var/tmp/rpm-tmp.5tT1ZE: line 29: fg: no job control

Reply:
ok i think the new push fixes that issue. I stick to the way we configure mariadb in common-rdo.